wandb/wandb_torch.py

```diff
@@ -302,6 +299,18 @@ def _torch_hook_handle_is_valid(self, handle):
         else:
             return handle.id in d
 
+    def _no_finite_values(self, tensor):
```

Comment:
could you add type annotations

Reply:
It seems like it's tricky to add type annotations here, because the type is torch.Tensor, and torch is imported with util.get_module instead of an import statement in the file.